setup.py

```diff
@@ -11,5 +11,6 @@
     packages=find_packages(exclude=['tests']),
     install_requires=[
         'python_moztelemetry',  # TODO: pin version
+        'click'
```

Comment:
Should we add an [entry point](http://click.pocoo.org/5/setuptools/) to setup.py?